### ISSUE TITLE
Endless horizontal scrolling

### ISSUE THREAD

**Issue by hosney** [olifolkerd/tabulator#1247]
When I have frozen columns on the right and the height is a just a little bit less than the content height the horizontal scroll bar seems to go on forever (The left position of the frozen cell increases with scroll)

Sample script to reproduce problem:

```
$("#example-table").tabulator({
  height: 270,
  columns: [
    { title: "Column 1", field: "col1", width: 500 },
    { title: "Column 2", field: "col2", width: 500 },
    { title: "Column 3", field: "col3", width: 500 },
    { title: "Column 4", field: "col4", width: 500 },
    { title: "X", field: "x", align: "center", frozen: true, width: 30 },
    { title: "Y", field: "y", align: "center", frozen: true, width: 30 }
  ]
});

var tabledata = [];
for (let i = 0; i < 10; i++) {
  tabledata.push({
    col1: "column 1 data",
    col2: "column 2 data",
    col3: "column 3 data",
    col4: "column 4 data",
    x: "X",
    y: "Y"
  });
}

$("#example-table").tabulator("setData", tabledata);
```

Pen: https://codepen.io/anon/pen/yErvrQ

**Comment by olifolkerd** [comment]
Hey @hosney 

That is definitely a bug you have found there! i will include a fix in the 4.0 release next month,

Thanks for letting me know.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @hosney 

I have pushed a fix for this to the 4.0 branch and will include it in the next release

Cheers

Oli :)
